2022-07_sprint/sprint_parties/index.md

```diff
@@ -1,8 +1,185 @@
+
+Sandra Meneses
+:::
+:::{grid-item-card}
+:link: https://www.linkedin.com/in/sandy-weng-a0959762/
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Sandy Weng" src="../../_static/sandy.jpg" class="rounded-circle">
+
+Sandy Weng
+:::
+::::
+
+### PyMC Organizers
+
+::::{grid} 2 2 3 5
+:gutter: 3
+
+:::{grid-item-card}
+:link: TODO
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Christian Luhmann" class="rounded-circle" src="../../_static/christian.jpg">
+
+Christian Luhmann
+:::
+:::{grid-item-card}
+:link: https://reshamas.github.io
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Reshama Shaikh" src="../../_static/reshama.jpg" class="rounded-circle">
+
+Reshama Shaikh
+:::
+:::{grid-item-card}
+:link: https://oriolabrilpla.cat
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Oriol Abril Pla" src="../../_static/oriol.jpg" class="rounded-circle">
+
+Oriol Abril Pla
+:::
+:::{grid-item-card}
+:link: https://twiecki.io/
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Thomas Wiecki" src="../../_static/thomas.jpg" class="rounded-circle">
+
+Thomas Wiecki
+:::
+::::
+
+### PyMC Mentors
+
+::::{grid} 2 2 3 5
+:gutter: 3
+
+:::{grid-item-card}
+:link: TODO
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Christian Luhmann" class="rounded-circle" src="../../_static/christian.jpg">
+
+Christian Luhmann
+:::
+:::{grid-item-card}
+:link: https://twiecki.io/
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Thomas Wiecki" src="../../_static/thomas.jpg" class="rounded-circle">
+
+Thomas Wiecki
+:::
+::::
+
+### About PyMC and Data Umbrella
+
+[Data Umbrella](https://www.dataumbrella.org/) is a global community for underrepresented persons in data science. It is a fiscally hosted project of Open Collective, a registered 501(c)(3) non-profit based in California, USA. Data Umbrella: organizes online speaker series on data science and open source, organizes sprints / hackathons, curates resources on inclusive practices. All levels are welcome, beginners and experts.
+
+[PyMC](https://www.pymc.io) is a probabilistic programming package for Python that allows users to fit Bayesian models using a variety of numerical methods, most notably Markov chain Monte Carlo (MCMC) and variational inference (VI). Its flexibility and extensibility make it applicable to a large suite of problems. PyMC is a non-profit project under NumFOCUS.
 
 ## Community partners
 
+::::{grid} 1 1 2 2
+:::{grid-item}
+![Python Ghana logo](https://www.pythonghana.org/static/assets/img/ug/about.png)
+:::
+:::{grid-item}
+**Python Ghana**
+
+TODO: Add short description
+:::
+:::{grid-item}
+![PyLadies Ghana logo](https://ghana.pyladies.com/static/assets/images/pyladies.png)
+:::
+:::{grid-item}
+**PyLadies Ghana**
+
+TODO: Add short description
+:::
+::::
+
 ## Sponsors
```

Comment:
By the way @reshamas how are the companies that qualify as sponsors chosen?

Reply:
Updated: https://pymc-data-umbrella--54.org.readthedocs.build/en/54/2022-07_sprint/sprint_parties/index.html#community-partners. Links and descriptions still missing, but that is easy to update once we have the content

Reply:
The logos look good!

I will add the links and descriptions later.  Thanks!